tools/CMakeLists.txt

```diff
@@ -96,7 +96,7 @@ endif()
 
 
 #openmp with clang smaller version 10 and user defined reductions is buggy https://bugs.llvm.org/show_bug.cgi?id=44134
-if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 10.0)
+if (CMAKE_CXX_COMPILER MATCHES "clang" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 10.0)
```

Comment:
Why that change?

Reply:
To not disable OpenMP when `icpx` is detected with `CMAKE_CXX_COMPILER_ID = Clang`. 
(I never understood the logic behind this disabling to be fair.)

Reply:
Isn’t the ID “Clang” for both?

Reply:
Actually I think the second part is wrong it should be “VERSION_LESS”.

Reply:
Thinking about this more, I feel like clang-9 and older is a bit outdated, how about we drop that block all together?

Reply:
I fixed it.

Clang-10 was released on 24 Mar 2020, so let's keep the workaround for now.